source/bindbc/imgui/dynload.d

```diff
@@ -34,7 +34,7 @@ bool loadImGuiSupport()
         ];
     }
     else version(Posix) {
-        const(char)[][2] libNames = [
+        const(char)[][1] libNames = [
```

Comment:
Similarly, part of the preamble, corresponding line is here: 
https://github.com/Inochi2D/bindbc-imgui/blob/main/generator/generator.d#L105